src/codec.rs

```diff
@@ -552,6 +552,55 @@ impl<T: Decode> Decode for Option<T> {
 	}
 }
 
+const _: () = {
+	use core::num::{
+		NonZeroI8,
+		NonZeroI16,
+		NonZeroI32,
+		NonZeroI64,
+		NonZeroI128,
+		NonZeroU8,
+		NonZeroU16,
+		NonZeroU32,
+		NonZeroU64,
+		NonZeroU128,
+	};
+	macro_rules! impl_for_non_zero {
+		( $( ($name:ty, $wrapped:ty) ),* $(,)? ) => {
```

Comment:
`wrapped` is not required.

Reply:
True, but also way less explicit. I prefer being explicit in macros.

Reply:
done